File: VIS360.Core/Entities/UserInfo.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace VIS360.Core.Entities
{
    public class UserInfo
    {
        public int ID { get; set; }
        
        public string Name { get; set; }
        public string Surname { get; set; }
        public string Email { get; set; }
        public int PhoneNumber { get; set; }

        public virtual User User { get; set; }

        
    }
}

File: android/app/src/main/java/com/example/ikesios/UserInfo.java
package com.example.ikesios;

public class UserInfo {

    public int ID;
    public String Name;
    public String Surname;
    public String Email;
    public int PhoneNumber;
    public  User User ;

    public int getID() {
        return ID;
    }

    public void setID(int ID) {
        this.ID = ID;
    }

    public String getName() {
        return Name;
    }

    public void setName(String name) {
        Name = name;
    }

    public String getSurname() {
        return Surname;
    }

    public void setSurname(String surname) {
        Surname = surname;
    }

    public String getEmail() {
        return Email;
    }

    public void setEmail(String email) {
        Email = email;
    }

    public int getPhoneNumber() {
        return PhoneNumber;
    }

    public void setPhoneNumber(int phoneNumber) {
        PhoneNumber = phoneNumber;
    }

    public User getUser() {
        return User;
    }

    public void setUser(User user) {
        User = user;
    }
}
